misc/Webscraper.user.js

```diff
@@ -0,0 +1,243 @@
+// ==UserScript==
+// @name         Web Scraper Helper
+// @namespace    http://coveo.github.io/
+// @version      0.2
+// @description  Web scraper helper tool for Coveo
+// @author       Jerome Devost
+// @grant        GM_addStyle
+//
+// @match        http://blog.coveo.com/*
+//
+// ==/UserScript==
+
+/* jslint -W109 */
+/* globals GM_addStyle */
+
+//
+// TO DO:
+// 1. Edit the @match above  (be mindful of http: vs https:). You can have multiple @match.
+// 2. Edit your Web Scraper Rules here
+//
+let jsonRules = [{
+  "for": {
+      "urls":[".*"]
+  },
+  "exclude": [{
+    "type": "CSS", "path": "body > header, iframe"
+    }, {
+    "type": "CSS", "path": ".main .wrapper .container aside, .main-content .featured-post .addthis_toolbox, div#disqus_thread"
+    }, {
+    "type": "XPATH", "path": "/html/body/footer"
+    }, {
+    "type": "XPATH", "path": "/html/body/div[3]"
+    }, {
+    "type": "CSS", "path": "div#posts div#placement-bottom, div#posts div#no-posts, div#posts div#form, div#posts div#footer, div#disqus_thread #dsq-comments .pingback"
+  }],
+  "metadata": {
+    "author": {
+      "type": "XPATH", "path": "/html/body//article//div[@class='meta-info']//div[@class='author-container']/span[@class='author']/a/text()"
+    },
+    "authorloginname": {
+      "type": "XPATH", "path": "substring-before(substring-after(/html/body//article//div[@class='meta-info']//div[@class='author-container']/span[@class='author']/a/@href,'/author/'),'/')"
+    },
+    "date": {
+      "type": "XPATH", "path": "/html/body//article//div[@class='meta-info']//div[@class='time-container']/time//@datetime"
+    },
+    "tags": {
+      "type": "XPATH", "path": "/html/body//article//div[@class='topics']/ul/li/a/text()"
+    },
+    "title": {
+      "type": "XPATH", "path": "/html/body//div[@class='title-container']/h2/a/text()"
+    }
+  }
+}];
+
+
+
+//
+// That's it, Reload the page you want to scrape. Rinse and Repeat.
+// Don't need to edit the code below this line.
+// Enjoy!
+// -----------------------------------------------------------------
+//
+
+
+
+
+
+(function() {
+'use strict';
```

Comment:
Indentation of the first section is messy, it would help readability to format it properly.